src/interpreter.rs

```diff
@@ -52,7 +56,7 @@ mod tests {
 
     fn fixture_plus() -> Word<i32> {
         Word {
-            entry: Box::new(move |i| {
+            entry: Box::new(|ref mut i| {
                 if let (Some(b), Some(a)) = (i.pop(), i.pop()) {
```

Comment:
Here, fixture does not need to move anything from its context. But it needs to manipulate a mutable reference to the interpreter